### PR TITLE
(content) Update label on transaction edit

### PR DESCRIPTION
**Pull request by mec**
This has been changed on the table view and this should match.

### CODE REVIEW

**Comment by mec** [comment]
[Improve "create programme" content - purpose of programme and value](https://trello.com/c/rI0cKhqL/674-improve-create-programme-content-purpose-of-programme-and-value)

File: config/locales/models/transaction.en.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ en:
         providing_organisation_type: Providing organisation type
         receiving_organisation_name: Receiving organisation name
         receiving_organisation_type: Receiving organisation type
+        value: Transaction amount
```
Comment (lozette) [inline config/locales/models/transaction.en.yml:22]: Where does this get used? Is the label automagically picked out, as I can't find it in any of the views?

Comment (lozette) [reply config/locales/models/transaction.en.yml:22]: Also on line 12 of this file is `curreny: Currency` (misspelling of the key). I know this isn't in this PR but I'm concerned that this key is misspelled!

Comment (mec) [reply config/locales/models/transaction.en.yml:22]: Yep, automagic. We have configured the GOVUK form builder to use `form.label.activity` see:

https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/ffe685f25e24b55b8e6c07686c41a0a6571e8def/config/initializers/form_builder.rb#L10

Comment (mec) [reply config/locales/models/transaction.en.yml:22]: No  probs on the typo - I'll fix it…

Comment (lozette) [reply config/locales/models/transaction.en.yml:22]: Love a bit of magic

**Pull request merged**